webpack/private/webpack_devserver.bzl

```diff
@@ -2,13 +2,21 @@
 
 load("@aspect_rules_js//js:defs.bzl", "js_run_devserver")
 
+load(":webpack_bundle.bzl", _create_configs = "create_configs")
+
 def webpack_devserver(
         name,
-        webpack_config,
+        srcs = [],
```

Comment:
Lets remove `srcs` and `deps` and just go with `data` since that the pattern for binary rules.